transcendence-app/test/auth/auth.e2e-spec.ts

```diff
@@ -25,19 +30,44 @@ describe('Auth Controller', () => {
       controllers: [AuthController],
     }).compile();
 
-    config = module.get<ConfigService<OAuth42Config>>(ConfigService);
-    app = module.createNestApplication();
+    app = moduleFixture.createNestApplication();
+    config =
+      moduleFixture.get<ConfigService<EnvironmentVariables>>(ConfigService);
+
+    const MemcachedStore = ConnectMemcached(session);
+    const memcachedHost = config.get('MEMCACHED_HOST') as string;
+    const memcachedPort = config.get('MEMCACHED_PORT') as string;
+
+    app.use(
+      session({
+        secret: config.get('SESSION_SECRET') as string,
+        resave: false,
+        saveUninitialized: false,
+        store: new MemcachedStore({
+          hosts: [`${memcachedHost}:${memcachedPort}`],
+          secret: config.get('MEMCACHED_SECRET'),
+        }),
+      }),
+    );
+    app.use(passport.initialize());
+    app.use(passport.session());
     app.init();
   });
 
-  it('should redirect to 42 oauth page', async () => {
+  it('Login [GET /login]', async () => {
     const response = await request(app.getHttpServer())
       .get('/auth/login')
       .expect(HttpStatus.FOUND);
     const authUrl = config.get('FORTYTWO_APP_AUTHORIZATION_URL');
     expect(response.headers['location']).toMatch(new RegExp(`^${authUrl}?`));
   });
 
+  it('Logout [DELETE /logout]', async () => {
```

Comment:
Why do we get forbidden when we attempt to log out here?

Reply:
We added the authenticated guard to this route in #115.
Now unauthenticated users accessing this route will get a Forbidden(403) response.

This test makes an HTTP request without valid cookies (as an unauthenticated user), hence the Forbidden response.